.github/workflows/build.yml

```
@@ -0,0 +1,162 @@
+        with:
+          args: " --file infrastructure/docker/Dockerfile --build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g) --tag khulnasoft/bw ."
+          cache_key: "${{ hashFiles('infrastructure/docker/Dockerfile') }}"
+
+      - name: Stop services
+        run: |
+          sudo service mysql stop || true
+          sudo service postgresql stop || true
+
+      - name: Run tests if needed
+        if: ${{ env.RUN_TESTS }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:1:81: Quote this to prevent word splitting

(shellcheck)